packages/query-broadcast-client-experimental/src/index.test.ts

```diff
@@ -0,0 +1,21 @@
+import { QueryClient, QueryCache } from '@tanstack/query-core'
+import { describe, expect, it, beforeEach } from 'vitest'
+import { broadcastQueryClient } from '.'
+
+describe('broadcastQueryClient', () => {
+  let queryClient: QueryClient
+  let queryCache: QueryCache
+
+  beforeEach(() => {
+    queryClient = new QueryClient()
+    queryCache = queryClient.getQueryCache()
+  })
+
+  it('should subscribe to the query cache', async () => {
+    broadcastQueryClient({
```

Comment:
I think we are generally missing a cleanup function here that should be returned from `broadcastQueryClient`. It should:

- call `unsubscribe` returned from `queryClient.getQuryCache().subscribe`, otherwise, we are leaking a subscription
- call [channel.close()](https://developer.mozilla.org/en-US/docs/Web/API/BroadcastChannel/close) on the created `BroadcastChannel`, otherwise, it can’t be garbage collected

then of course the test should call that, too.